cmd/katalyst-agent/app/options/qrm/cpu_plugin.go

```diff
@@ -23,24 +23,28 @@ import (
 )
 
 type CPUOptions struct {
-	PolicyName                string
-	EnableCPUAdvisor          bool
-	ReservedCPUCores          int
-	SkipCPUStateCorruption    bool
-	EnableCPUPressureEviction bool
-	EnableSyncingCPUIdle      bool
-	EnableCPUIdle             bool
+	PolicyName                 string
+	EnableCPUAdvisor           bool
+	ReservedCPUCores           int
+	SkipCPUStateCorruption     bool
+	EnableCPUPressureEviction  bool
+	EnableSyncingCPUIdle       bool
+	EnableCPUIdle              bool
+	EnableFullPhysicalCPUsOnly bool
+	CPUAllocationOption        string
 }
 
 func NewCPUOptions() *CPUOptions {
 	return &CPUOptions{
-		PolicyName:                "dynamic",
-		EnableCPUAdvisor:          false,
-		ReservedCPUCores:          0,
-		SkipCPUStateCorruption:    false,
-		EnableCPUPressureEviction: false,
-		EnableSyncingCPUIdle:      false,
-		EnableCPUIdle:             false,
+		PolicyName:                 "dynamic",
+		EnableCPUAdvisor:           false,
+		ReservedCPUCores:           0,
+		SkipCPUStateCorruption:     false,
+		EnableCPUPressureEviction:  false,
+		EnableSyncingCPUIdle:       false,
+		EnableCPUIdle:              false,
```

Comment:
maybe it's better to separate options/configs for different policies, ie.
1. define general options/configs in `CPUOptions/CPUConfigurations`
2. define native-policy-specified  options/configs in `CPUNativePolicyOptions/CPUNativePolicyConfigurations`
3. define dynamic-policy-specified  options/configs in `CPUDynamicPolicyOptions/CPUDynamicPolicyConfiguration`

Reply:
done